kvbc/include/ror_app_state.hpp

```diff
@@ -0,0 +1,65 @@
+#pragma once
+
+#include <memory>
+#include <string>
+#include "bcstatetransfer/SimpleBCStateTransfer.hpp"
+#include "blockchain/block.h"
+#include "blockchain/db_adapter.h"
+#include "kv_types.hpp"
+#include "sliver.hpp"
+#include "status.hpp"
+#include "storage/db_interface.h"
+#include "object_store/object_store_client.hpp"
+
+const std::string kLastBlockIdKey = "1";
+const std::string kLastReachableBlockIdKey = "2";
```

Comment:
Can we declare these as either `static const...` or better yet `inline const...`?